src/components/transactions/TxDetails/TxData/index.tsx

```diff
@@ -82,6 +83,10 @@ const TxData = ({
     return <OnChainConfirmation data={txDetails.txData} />
   }
 
+  if (isExecTxData(txDetails.txData)) {
+    return <ExecTransaction data={txDetails.txData} />
+  }
```

Comment:
Have you considered how this will affect all other `execTransaction` details?

Reply:
Which other details?

Reply:
As per discussion: we always display the transaction executed so the `execTransaction` here is of the nested Safe.